openvswitch/2.17.1/openvswitch-2.17.1.patch

```diff
@@ -0,0 +1,252 @@
+diff --git a/Documentation/intro/install/general.rst b/Documentation/intro/install/general.rst
+index a297aadac..745ca4aea 100644
+--- a/Documentation/intro/install/general.rst
++++ b/Documentation/intro/install/general.rst
+@@ -85,6 +85,10 @@ need the following software:
+   an OpenFlow controller. If libssl is installed, then Open vSwitch will
+   automatically build with support for it.
+ 
++- libwolfssl, from wolfSSL, is optional and can be used instead of openssl to
++  provide commercial open source library for TLS and Crypto. Enabled using
```

Comment:
I'd probably change this to "provide commercial and open source libraries". Can maybe say "as an alternative to" instead of "instead of".